libmamba/src/core/solver.cpp

```diff
@@ -598,6 +602,36 @@ namespace mamba
         return problems.str();
     }
 
+    std::ostream& MSolver::explain_problems(std::ostream& out) const
+    {
+        bool experimental = Context::instance().experimental_sat_error_message;
+        if (experimental)
+        {
+            fmt::print(out, "{:=^80}\n", " Experimental satisfiability error messages ");
```

Comment:
Did you check there there will not be any issues of mixing spdlog's output with `fmt::print`? they use the same output?

Reply:
I think they flush at different moments so the outputs are not intertwined.
Here the `out` is a parameter so this is managed outside this function.